cdap-ui/app/features/search/templates/search-list-view.html

```diff
@@ -0,0 +1,55 @@
+<!--
+  Copyright © 2015 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<my-breadcrumbs></my-breadcrumbs>
+<div class="row">
+  <div class="col-xs-12">
+    <div class="row">
```

Comment:
instead of nesting a col-xs-12 here, we can just simply make this section a row, and the next block another row